crates/re_renderer/src/resource_managers/texture_manager.rs

```diff
@@ -87,11 +97,12 @@ impl TextureManager2D {
             white_texture,
             device,
             queue,
+            texture_cache: Default::default(),
+            accessed_textures: Default::default(),
         }
     }
 
     /// Creates a new 2D texture resource and schedules data upload to the GPU.
-    #[allow(clippy::unused_self)]
     pub fn create(
```

Comment:
plz add a todo to remove this - we should always use the `get_or_create` in here since anything that doesn't have data should go to the `texture_pool` directly
this would fix situations we have right now where a mesh loads the same texture twice